src/App.js

```diff
@@ -1,26 +1,10 @@
-import logo from './logo.svg';
-import './App.css';
+import Calculator from './components/calculator';
 
 function App() {
   return (
     <div className="App">
```

Comment:
- [ ] The `App.js` component should be class-based. Please update it.

Reply:
HI dear @Dipeshtwis I think that this is not a requirement, they said that all components should be class-based but I don't know if App.js is also a component (they said that in the App.js we should only import the calculator component), anyway it's done.